openmc/surface.py

```diff
@@ -185,22 +184,22 @@ class Plane(Surface):
 
     Parameters
     ----------
-    surface_id : int
+    surface_id : int, optional
         Unique identifier for the surface. If not specified, an identifier will
         automatically be assigned.
-    boundary_type : {'transmission, 'vacuum', 'reflective', 'periodic'}
+    boundary_type : {'transmission, 'vacuum', 'reflective', 'periodic'}, optional
```

Comment:
Wait, do we have periodic BCs?? They aren't included in the docs [here](http://mit-crpg.github.io/openmc/usersguide/input.html#surface-element) on the XML input. If we do have them that is awesome and super useful!